<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QWidget" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>238</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Informazioni sul nuovo dipinto</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="scuola_lineEdit">
       <property name="placeholderText">
        <string>Scuola...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="autore_lineEdit">
       <property name="placeholderText">
        <string>Autore...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titolo_lineEdit">
       <property name="placeholderText">
        <string>Titolo...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="data_lineEdit">
       <property name="placeholderText">
        <string>Data...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sala_lineEdit">
       <property name="placeholderText">
        <string>Sala...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Annulla</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="confirmButton">
         <property name="text">
          <string>Conferma</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
